config_test.go

```diff
@@ -4,9 +4,33 @@ import (
 	"path/filepath"
 	"testing"
 
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/send"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
 )
 
+const (
+	testDir      = "config_test"
+	testSettings = "evg_settings.yml"
+)
+
+// TestConfig creates test settings from a test config.
+func testConfig() *Settings {
```

Comment:
is there a test for the fallback default behavior?

Reply:
added a test